frontend/src/components/common/SimpleTable.tsx

```diff
@@ -71,12 +101,17 @@ export default function SimpleTable(props: SimpleTableProps) {
     filterFunction = null,
     emptyMessage = null,
     errorMessage = null,
+    defaultSortingColumn
   } = props;
   const [page, setPage] = React.useState(0);
   const [currentData, setCurrentData] = React.useState(data);
+  const [displayData, setDisplayData] = React.useState(data);
   const rowsPerPageOptions = props.rowsPerPage || [5, 10, 50];
   const [rowsPerPage, setRowsPerPage] = React.useState(rowsPerPageOptions[0]);
   const classes = useTableStyle();
+  const [isIncreasingOrder, setIsIncreasingOrder] = React.useState(!!(defaultSortingColumn &&
+    defaultSortingColumn > 0));
+  const [sortColIndex, setSortColIndex] = React.useState(0);
```

Comment:
By default we shouldn't sort anything (so we keep the current behavior of just showing whatever the data has).
The way I understand this logic is that we have this `sortColIndex` but also `defaultSortingColumn` and then we use the latter once below, and the other if the user clicks it.

This think we can deeply simplify this by using just the `sortColIndex` that will take the value of `defaultSortingColumn` (or `defaultSortingColumn != 0 : abs(defaultSortingColumn) - 1 : null` if we keep a separate `isIncreasingOrder` state for the order alone, being `null` the "don't sort anything" state).

Reply:
So this is where our onMount effect is helpful in that we check whether there is a defaultSortingColumn and if not don't do anything with data(don't sort)